openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
I see a few options:
1) Don't support a TransportXS which doesn't use nu-scattering.  That doesn't seem so bad since its not technically correct to be using normal scattering matrices vice nu-scatter matrices.
2) If we don't want to go that route, your NuTransportXS class idea seems good for now, though like I said above, that might explode with future corrections too.
3) In the long-term (or now if it seems trivial enough), what about simplifying this considerably by recognizing that all transport corrections work by including a deltaSigma_tr which is applied to a total xs and the scattering matrix diagonal.  What if we just have an extra MGXS class for deltaSigma_tr? Then, Library can know how to apply it with some `apply_correction()` function, or we can leave it to the downstream code to do something with.

Thoughts @wbinventor or @paulromano or anyone else who has thought about such corrections before?